README.md

```diff
@@ -148,8 +148,9 @@ To release a new version of `datadog-ci`:
 3. Push the branch along with the tag to the upstream (GitHub) with `git push --tags origin name-of-the-branch`, create a Pull Request with the changes introduced in the description details, and get at least one approval. For example, see this [sample pull request](https://github.com/DataDog/datadog-ci/pull/78).
 4. Merge the Pull Request.
 5. Create a GitHub Release from the [Tags page](https://github.com/DataDog/datadog-ci/tags) with the description of changes introduced.
-6. Once the release has been created, a GitHub Action publishes the package, and a Gitlab pipeline publishes the Docker image. Make sure these jobs succeed.
-7. If the release introduced some **changes in the** `synthetics` **command**, you have to upgrade `datadog-ci` in the following projects:
+6. Once the release has been created, a GitHub Action publishes the package. Make sure the job succeeds.
+7. Once the package has been published, [go to Gitlab](https://gitlab.ddbuild.io/DataDog/datadog-ci/-/pipelines), find the pipeline for your tag and run the Docker image build jobs by pressing the "play" button on the `build` stage. Once the jobs pass, the `release` stage will automatically trigger. Make sure all the jobs succeed.
```

Comment:
```suggestion
7. Once the package has been published, [go to GitLab](https://gitlab.ddbuild.io/DataDog/datadog-ci/-/pipelines), find the pipeline for your tag, and press the **play** button on the `build` stage to run the Docker image build jobs. Once the jobs pass, the `release` stage automatically triggers. Make sure all the jobs succeed.
```

Reply:
In addition to the above, could you confirm that the link in use here is intended to be shared? It looks like it may be an internal link.

Reply:
It's only accessible from within Datadog, but it is already public in the datadog-agent repo and others.